docs/userguide/cicd/working-with-pipecd.md

```diff
@@ -0,0 +1,139 @@
+---
+title: Working with PipeCD
+---
+
+This topic walks you through how to use the [PipeCD](https://github.com/pipe-cd/pipecd/) to manage your workload
+`across clusters` with `Karmada`.
+
+## Prerequisites
+### PipeCD Installation
+
+For the purpose of this example, you can install PipeCD follow the [Quickstart](https://pipecd.dev/docs/quickstart/) instructions.
+
+You can access to `localhost:8080`, login and get the following UI to confirm PipeCD is ready.
+
+![](../../resources/userguide/cicd/pipecd/pipecd-ready-page.png)
+
+### Karmada Installation
+In this example, we are using a Karmada environment with at least `3` member clusters joined.
+
+You can set up the environment by `hack/local-up-karmada.sh`, which is also used to run our E2E cases.
+
+```bash
+$ kubectl get clusters
+NAME      VERSION   MODE   READY   AGE
+member1   v1.19.1   Push   True    18h
+member2   v1.19.1   Push   True    18h
+member3   v1.19.1   Pull   True    17h
+```
+
+## Install and configure PipeCD agent (piped) to make it works with Karmada
+
+This step install the PipeCD agent `piped` and configure it to connect the Karmada API server.
+
+In this example, for simplicity we follow the instruction at [Install piped on a single machine](https://pipecd.dev/docs/installation/install-piped/installing-on-single-machine/). The piped will be installed in your local machine, and use the `karmada-apiserver` context to connect with the Karmada control plane.
+
+Access [localhost:8080/settings/piped?project=quickstart](http://localhost:8080/settings/piped?project=quickstart) from your browser, and register information for a new piped.
+
+Prepare piped configuration yaml as below and save it as `piped-config.yaml`
+
+```yaml
+apiVersion: pipecd.dev/v1beta1
+kind: Piped
+spec:
+  projectID: quickstart
+  pipedID: <YOUR_PIPED_ID>
+  pipedKeyData: <YOUR_PIPED_KEY_DATA>
```

Comment:
Do you want to explain how to set the `YOUR_PIPED_ID` and `YOUR_PIPED_KEY_DATA` a little bit?

Reply:
I referenced the quickstart docs for pipecd installation in the previous section already, so I think users are already know with it us up to this place. wdyt? 🤔

Reply:
Why not just update it to `test` as you give an example with that ID.

Reply:
Since that place requires not the piped name but the piped ID, which is different depending on users in the registered time, I think it could be less confusing if we keep it as is instead of mentioning the `test` piped name here. Users would also know that they need to find their `<YOUR_PIPED_ID>` and `<YOUR_PIPED_KEY_DATA>`, which could be found if they follow the quickstart docs.

Reply:
I can add something like

```md
For <YOUR_PIPED_ID> and <YOUR_PIPED_KEY_DATA>, please refer to [installing piped](https://pipecd.dev/docs-v0.48.x/quickstart/#12-installing-piped) docs.
```

WDYT? 🤔

Reply:
Much better now. Thanks.

Reply:
Addressed by https://github.com/karmada-io/website/pull/680/commits/0fdbb787e3df90b5febe1164050b7f996497b3c8 🙏